microsoft/botframework-solutions
# [Skill Template][TypeScript] Create Skill Template project structure

## Description
- Update `Skill-Template` folder structure (C# and TS)
- Create TS project structure 
- Add simple logic to the generator
- Implement **TSLint**
- Add Skill Template TS documentation

Related to #941 

## Testing Steps
1. Go to `AI/templates/Skill-Template/src/typescript/generator-botbuilder-skill/generators/app/templates`
2. Check the project structure, should be similar to C# solution

## Checklist
